src/expression.ts

```diff
@@ -529,14 +534,14 @@ export class YieldExpr extends BaseExpr<NodeKind.YieldExpr> {
     /**
      * The expression to yield (or delegate) to.
      */
-    readonly expr: Expr | undefined,
+    readonly expr: Expr,
```

Comment:
why? 

```ts
    export interface YieldExpression extends Expression {
        readonly kind: SyntaxKind.YieldExpression;
        readonly asteriskToken?: AsteriskToken;
        readonly expression?: Expression;
    }
```


```ts
function* generator () {
   yield;
}
```

> Defines the value to return from the generator function via [the iterator protocol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#the_iterator_protocol). If omitted, undefined is returned instead.

Reply:
Hmm I thought I checked that. Ok will reverse